### ISSUE TITLE
Add docstrings

### ISSUE THREAD

**Issue by scottclowe** [hadim/read-roi#12]
It would be great if you could add docstrings to `read_roi_file` and `read_roi_zip`. That would make it a lot easier to work with the package.

**Comment by hadim** [comment]
I agree with you and thanks for opening the issue. Will try to do that at some points.

**Comment by dipanjanbhattacharya** [comment]
Hi ,

i faced with this error in opening zip roi file

_**UnboundLocalError: local variable 'channel' referenced before assignment**_

**Comment by scottclowe** [comment]
Hi, @dipanjanbhattacharya.
Could you please:
1. Open a new issue (https://github.com/hadim/read-roi/issues/new) to report this
2. Provide the full stack trace for the error, showing line numbers.
3. If you could also share a copy of the ROI file you are trying to open, that would help a lot.
